modules/developers-guide/pages/tutorials/define-an-actor.adoc

```diff
@@ -79,39 +79,39 @@ cd src/actor_hello
 ----
 . Open the template `+main.mo+` file in a text editor and delete the existing content.
 +
-The next step is to write a program that prints a statement like the traditional "Hello, World!" sample program.
-To compile the program for the {IC}, however, your program must include an `+actor+` object with a `+public+` function.
+The next step is to write a canister that prints a statement like the traditional "Hello, World!" sample canister.
+To compile the canister for the {IC}, however, your canister must include an `+actor+` object with a `+public+` function.
 . Copy and paste the following sample code into the `+main.mo+` file:
 +
 [source.copy,motoko,no-repl]
 ----
 include::example$actor_hello.mo[]
 ----
 +
-Let's take a closer look at this simple program:
+Let's take a closer look at this simple canister:
 +
 --
-* The program imports a `+Debug+` module to provide the `+print+` functionality.
-* The program uses the `+public query func+` to define a query method because, in this case, the `+actor_hello+` program doesn't make any changes to the state of the canister or perform any operations that would update the data you are accessing.
+* The canister imports a `+Debug+` module to provide the `+print+` functionality.
+* The canister uses the `+public query func+` to define a query method because, in this case, the `+actor_hello+` canister doesn't make any persistent changes to the state of the canister smart contract - any changes are transient and discarded after the query completes.
 --
 +
 For more information about using a query call, see link:../concepts/canisters-code{outfilesuffix}#query-update[query calls] in link:../concepts/canisters-code{outfilesuffix}#canister-state[Canisters include both program and state].
 . Save your changes and close the `+main.mo+` file.
 
-== Build the program with a local identifier
+== Build the canister with a local identifier
 
-You are probably only going to use this simple program for some local testing.
-Therefore, there's no need to reserve a unique canister identifier on the {IC} network to hold the compiled output for the program.
+You are probably only going to use this simple canister for some local testing.
+Therefore, there's no need to reserve a unique canister identifier on the {IC} blockchain mainnet to hold the compiled output for the canister.
 
-In this scenario, you can compile the program without connecting to an {IC} network at all.
+In this scenario, you can compile the canister without connecting to the {IC} blockchain mainnet at all.
 Instead, the `+dfx build+` command creates a local, hard-coded canister identifier for you to use.
```

Comment:
```suggestion
Instead, the `+dfx build+` command creates a local, hard-coded canister identifier for you to use.
```
Why hard-coded?

Reply:
I don't know :-) I didn't write that text :-)

Reply:
I think that is even false. I flagged it with the SDK team.